streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractProcessorContext.java

```diff
@@ -113,10 +113,12 @@ public void register(final StateStore store,
     }
 
     /**
+     * @throws UnsupportedOperationException if the current task type is standby
```

Comment:
No logical changes here, just added a check for any methods that were previously overridden in the standby context to throw UnsupportedOperation